host_applications/linux/apps/raspicam/RaspiVid.c

```diff
@@ -2937,7 +2938,7 @@ int main(int argc, const char **argv)
                }
 
                // Send all the buffers to the splitter output port
-               if (state.raw_output) {
+               if (state.callback_data.raw_file_handle) {
                   int num = mmal_queue_length(state.splitter_pool->queue);
```

Comment:
Is the raw_output flag still requried elsewhere?

Reply:
I don't believe so.
It's used for setup, and nuked if the file handle fails to open. The rest uses the file handle.

Reply:
Should probably remove it then, but TBH, hardly worth the effort. Othewise LGTM.

Reply:
Sorry, misintepreted your comment.
The command line parser sets raw_output. The setup code then checks it and will open the file and assign callback_data.raw_file_handle.
The rest of the code then only looks at callback_data.raw_file_handle.